content/home.md

```diff
@@ -0,0 +1,42 @@
+type: page
+slug: home
+body_class: home
+title: Home
+---
+
+PyCon UK is the annual gathering of the UK Python community and its friends
+from around the world.  This year, the conference is held at [Cardiff City
+Hall](http://www.cardiffcityhall.com/), from Thursday 15th to Monday 19th
+September.
+
+We&rsquo;ll be putting on a programme of talks, workshops, and other events
```

Comment:
This is insanely picky of me, I realise, but this should arguably be `&apos;`, not `&rsquo;`.

Realistically, though, is it worth escaping all of these unicode chars?